src/apps/audioCompression.ts

```
@@ -0,0 +1,73 @@
+ * Generate a realistic-ish audio-like waveform with dynamic variation.
+ */
+export function makeSignal(n: number): Float32Array {
+  const s = new Float32Array(n);
+  const components = [
+    { f: 0.018, a: 0.5 }, { f: 0.035, a: 0.3 }, { f: 0.06, a: 0.18 },
+    { f: 0.011, a: 0.6 }, { f: 0.042, a: 0.22 }, { f: 0.089, a: 0.14 },
+    { f: 0.005, a: 0.4 },
+  ];
+  for (let i = 0; i < n; i++) {
+    let v = 0;
+    components.forEach(({ f, a }) => (v += Math.sin(i * f * Math.PI * 2) * a));
```

Biome (2.4.6)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)